utils/build/virtual_machine/weblogs/dotnet/provision_test-app-dotnet-unsupported.yml

```diff
@@ -0,0 +1,45 @@
+lang_variant:
+    name: dotnet
+    cache: true
+    version: 3
+    install:
+      - os_type: linux
+        os_distro: deb
+        os_branch: ubuntu20_amd64
+        remote-command: |
+          sudo apt-get update;
+          sudo apt-get install -y apt-transport-https;
+          sudo apt-get update;
+          sudo apt-get install -y dotnet-sdk-2.1
+
+      - os_type: linux
+        os_distro: deb
+        remote-command: |
+          sudo apt-get update;
+          sudo apt-get install -y apt-transport-https;
+          sudo apt-get update;
+          sudo apt-get install -y dotnet-sdk-2.1
+
+weblog:
+    name: test-app-dotnet-unsupported
+    #Ubuntu 21 (arm64) excluded due to there is not installable cadidate for dotnet-sdk-6.0
+    #Ubuntu 21 (amd64) excluded due to there is not installable cadidate for dotnet-sdk-6.0
+    #Centos 7. I can't find the libicu60 package for centos 7. Only available version 50.2
+    #Amazon Linux 2022: Incompatible runtime version installed. Not breaks app, but not instrumented
+    exact_os_branches: [Ubuntu20amd64]
```

Comment:
The correct branch name is "ubuntu20_amd64"
https://github.com/DataDog/system-tests/blob/86f272e8ccde244d37b544ca246f78eb229e6c4c/utils/_context/virtual_machines.py#L311